### DISCUSSION TITLE
关于洛雪音乐无法播放解决方法

### DISCUSSION THREAD

**Discussion by aectn** [lyswhut/lx-music-desktop#1688] [General]
https://github.com/OneCodeMonkey/music-sources
按照这个帖子的方法，在洛雪设置里面改

**Comment by aectn** [comment]
移动端把接口改成试听接口

**Comment by aectn** [comment]
https://www.sixyin.com/8498.html
这个是六音音源的原帖
下载：蓝奏云https://soso.lanzouj.com/b00p9c94f#8mno 密码8mno

**Comment by yuhuapang** [comment]
绷不住了，洛雪大佬在找工作
